tests/check_check_master.c

```diff
@@ -230,6 +230,8 @@ static master_test_t master_tests[] = {
   { "Simple Tests", "test_ck_assert_pstr_ne_with_null", CK_FAILURE, "Assertion 't != s' failed: t == (null), s == (null)" },
   { "Simple Tests", "test_ck_assert_ptr_eq", CK_FAILURE, "Assertion 'x == y' failed: x == 0x1, y == 0x2" },
   { "Simple Tests", "test_ck_assert_ptr_ne", CK_FAILURE, "Assertion 'x != z' failed: x == 0x1, z == 0x1" },
+  { "Simple Tests", "test_ck_assert_ptr_null", CK_FAILURE, "Assertion 'x == NULL' failed: x == 0x1" },
+  { "Simple Tests", "test_ck_assert_ptr_nonnull", CK_FAILURE, "Assertion 'x != NULL' failed: x == 0" },
```

Comment:
I thought about it. This assert belongs to `prt_*` macros. And they just print pointer value. So I think printing NULL has no sense. Another way all `ptr_*` macros could be changed to print `NULL` instead of zero. Anyway, it could be a separate commit.